src/main/java/ou/capstone/notams/ConnectToAPI.java

```
@@ -49,7 +49,14 @@ public static void main(String[] args) throws Exception {
 
         System.out.println("URL: " + uri);
         System.out.println("Status: " + response.statusCode());
-        System.out.println(response.body());
+        final String body = response.body();
+        System.out.println(body);
+
+        if (response.statusCode() == 200) {
+            new GeoJsonReader().readGeoJson(body);
+        } else {
+            System.err.println("Non-200 from FAA; skipping parse.");
```

Can we skip the parse and continue like normal? If not, this is an exceptional case and we should throw an Exception.

---

Updated ConnectToAPI on non-200, throws an IOException